test/parallel/test-wasm-simple.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+require('../common');
+
+const assert = require('assert');
+const fixtures = require('../common/fixtures');
+
+const buffer = fixtures.readSync('test.wasm');
+
+assert.ok(WebAssembly.validate(buffer), 'Buffer should be valid WebAssembly');
+
+WebAssembly.instantiate(buffer, {}).then((results) => {
+  assert.strictEqual(
+    results.instance.exports.addTwo(10, 20),
+    30,
+    'Exported function should add two numbers.',
```

Comment:
Suggestion: it would likely be better to either omit the message here or include the `expected` and `actual` values in the message.

Reply:
is this worth blocking on or should we land and fix in a follow up?

Reply:
yea, we should just land this